opcua_dbus.c

```diff
@@ -50,26 +61,56 @@ bool dbus_temp_init(void)
         return false;
     }
 
+    // dbusproxy_ports
+    dbusproxy_ports = g_dbus_proxy_new_for_bus_sync(
+        G_BUS_TYPE_SYSTEM,
+        G_DBUS_PROXY_FLAGS_NONE,
+        NULL,
+        PORTS_DBUS_SERVICE,
+        PORTS_DBUS_OBJECT,
+        PORTS_DBUS_INTERFACE,
+        NULL,
+        &error);
+    if (NULL == dbusproxy_ports)
+    {
+        LOG_E(
+            "%s/%s: Failed to create a proxy for accessing %s (%s)",
+            __FILE__,
+            __FUNCTION__,
+            PORTS_DBUS_OBJECT,
+            error->message);
+        g_error_free(error);
+        return false;
+    }
+
```

Comment:
I'd prefer if we add a helper function that takes a dbusproxy double pointer and the service/object/interface values as inparameter and returns a bool. Then this init_all would consist of one single return statement with return <call to first> && <call to second>.